lib/components/narrative/printable/printable-itinerary.js

```diff
@@ -92,19 +134,49 @@ class AccessLeg extends Component {
   }
 
   render () {
-    const { leg } = this.props
+    const { configCompanies, customIcons, leg } = this.props
+
+    // calculate leg mode label in a special way for this component
+    let legModeLabel = getLegModeLabel(leg)
+
+    if (leg.rentedBike) {
+      // FIXME: Special case for TriMet that needs to be refactored to
+      // incorporate actual company.
```

Comment:
I dislike writing new FIXMEs, but I figured this could be accomplished while refactoring the OTP vehicle rental updaters.